cmd/controller-bootstrap/command/root.go

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package command
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	appName      = "controller-bootstrap"
+	appShortDesc = "A bootstrap tool to initialize an ACK service controller repository"
+)
+
+var (
+	optServiceAlias       string
+	optRuntimeVersion     string
+	optAWSSDKGoVersion    string
+	optDryRun             bool
+	optExistingController bool
+	optOutputPath         string
+	optModelName          string
+)
+
+// rootCmd represents the base command when called without any subcommands
+// placeholder for cobra description
+var rootCmd = &cobra.Command{
+	Use:   appName,
+	Short: appShortDesc,
+}
+
+func init() {
+	rootCmd.PersistentFlags().StringVarP(
+		&optServiceAlias, "aws-service-alias", "s", "", "Supplied AWS service alias",
+	)
+	rootCmd.PersistentFlags().StringVarP(
+		&optRuntimeVersion, "ack-runtime-version", "r", "", "Version of aws-controllers-k8s/runtime",
+	)
+	rootCmd.PersistentFlags().StringVarP(
+		&optAWSSDKGoVersion, "aws-sdk-go-version", "v", "", "Version of github.com/aws/aws-sdk-go used to infer service metadata and resources",
+	)
+	rootCmd.PersistentFlags().BoolVarP(
+		&optDryRun, "dry-run", "d", false, "Optional: if true, output files to stdout",
+	)
+	rootCmd.PersistentFlags().BoolVarP(
+		&optExistingController, "existing-service-controller", "e", false, "Optional: if true, update the existing service controller",
+	)
```

Comment:
nit:
```suggestion
		&optExistingController, "existing-controller", "e", false, "Optional: if true, update the existing service controller",
	)
```

Reply:
The suggested changes have been addressed. Thank you.